static/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "midnight-static"
```

Comment:
Probably best to have this be `midnight-ledger-static`, this seems a bit too generic.